### DISCUSSION TITLE
画了个数据流程图，供大家参考

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#62] [2️⃣ 聊项目]
受 [github-explorer 项目](https://github.com/ClickHouse/github-explorer)，[OSS Insight 项目](https://github.com/pingcap/ossinsight)的参照，画了一个数据生产的流程图，还不完善，大家可以补充与讨论~

https://xlab2017.yuque.com/docs/share/1e0473fc-3a63-431a-9188-860c7974fbc2?# 《🍁 X-lab 数据生产流程图》

![image](https://user-images.githubusercontent.com/15010826/171318785-eecff6aa-f029-4ae4-86d4-7cc87085f301.png)

**Comment by xgdyp** [comment]
体验了下OSSinsight 超级赞！

**Reply by will-ww** [reply]
> 体验了下OSSinsight 超级赞！

是的，非常值得我们学习，既可以把我们的指标推广到他们的项目中，又可以用他们的可视化设计来丰富我们的项目，大家可以体会下，这种开源的方式所带来的好处~

**Reply by xgdyp** [reply]
补充下我之前看到cncf也有个类似的项目：https://devstats.cncf.io/ 也是grafana的

**Reply by will-ww** [reply]
Jimmysong的博文中也提了几款：https://jimmysong.io/blog/oss-insight/

**Reply by zhicheng-ning** [reply]
> Jimmysong的博文中也提了几款：https://jimmysong.io/blog/oss-insight/

Jimmysong！我最近就是在看他整理的云原生知识库

**Comment by zhicheng-ning** [comment]
typo：MangoDB -> MongoDB
